src/python/strelka/scanners/scan_transcode.py

```diff
@@ -0,0 +1,37 @@
+import io
+import logging
+
+import pillow_avif
+from PIL import Image
+from pillow_heif import register_heif_opener
+
+from strelka import strelka
+
+logging.getLogger("PIL").setLevel(logging.WARNING)
+
+# Must be imported as a plugin, doesn't need to be used
+_ = pillow_avif.AvifImagePlugin
+
+register_heif_opener()
+
+
+class ScanTranscode(strelka.Scanner):
+    """
+    Converts supported images to PNG for easier scanning
```

Comment:
Should we change the docstring here to "Converts supported images to JPG..." (as it is the default) or Converts supported images for easier scanning"?

Reply:
Fixed